CppCoreGuidelines.md

```diff
@@ -10385,7 +10385,7 @@ Readability and safety.
 
 As an optimization, you may want to reuse a buffer as a scratch pad, but even then prefer to limit the variable's scope as much as possible and be careful not to cause bugs from data left in a recycled buffer as this is a common source of security bugs.
 
-    {
+    void writeFile() {
```

Comment:
I guess you are right, I forgot to check that. For consistency, should the inner calls also be snake-case, then? In the following lines, I mean.